<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>model_select</class>
 <widget class="QMainWindow" name="model_select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QLineEdit,QPlainTextEdit,QSpinBox,QDoubleSpinBox{

border:1px solid  #C0DCF2;
padding:2px;
background:none;
selection-background-color:#EEEEEE;
selection-color:#000000;
}

QLineEdit[echoMode=&quot;2&quot;]{
lineedit-password-character:9679;
}

QPushButton
{
    color: rgb(255, 255, 255);
    background-color: rgb(85, 170, 255);
    border:none;
    padding: 3px;
    font-family: &quot;微软雅黑&quot;;
    font-size: 12px;
    text-align: center;
}
QComboBox,QDateEdit,QSpinBox,QDoubleSpinBox{
border-radius:3px;
padding:3px 5px 3px 5px;
border:1px solid #C0DCF2;
}

QComboBox::drop-down,QDateEdit::drop-down{
subcontrol-origin:padding;
subcontrol-position:top right;
width:15px;
border-left-width:1px;
border-left-style:solid;
border-top-right-radius:3px;
border-bottom-right-radius:3px;
border-left-color:#C0DCF2;
}

QComboBox::down-arrow,QDateEdit::down-arrow{
image:url(:/image/array_down.png);
}
.QGroupBox{
border:1px solid #C0DCF2;
border-radius:5px;
}

QRadioButton{
color:#386487;
spacing:2px;
}

QRadioButton::indicator{
width:15px;
height:15px;
}

QRadioButton::indicator::unchecked{
image:url(&quot;D:/Documents/PyQt5/qt_protein/qt_protein/ui/images/radio_normal.png&quot;);
}

QRadioButton::indicator::checked{
image:url(&quot;D:/Documents/PyQt5/qt_protein/qt_protein/ui/images/radio_selected.png&quot;);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>53</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模型选择</string>
    </property>
   </widget>
   <widget class="QComboBox" name="model_box">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>LightGBM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XGBoost</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>13</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>训练数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_file_button">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_path">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>11</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="params_setup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>331</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>参数设置</string>
    </property>
    <widget class="QPushButton" name="add_params_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>124</y>
       <width>75</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>添加参数</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="params_dict_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>311</width>
       <height>91</height>
      </rect>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="cross_validation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>263</y>
      <width>331</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>交叉验证</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>32</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>随机抽取</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="use_cross_validation">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>30</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>是</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="dontuse_cross_validation">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>否</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>61</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>随机数种子</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="random_seed">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>61</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>101</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>折数</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="n_flods">
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>96</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="output_dir">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>490</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>输出目录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tarin_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始训练</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>411</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>评价指标</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>407</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>pcc</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="output_path">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>487</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="output_dir_btn">
    <property name="geometry">
     <rect>
      <x>217</x>
      <y>487</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>迭代次数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="num_round">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>447</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
